pyminifier/pyminifier.py

```diff
@@ -420,9 +421,8 @@ def main():
                 "(https://github.com/liftoff/pyminifier)\n")
         # Either save the result to the output file or print it to stdout
         if options.outfile:
-            f = io.open(options.outfile, 'w', encoding='utf-8')
-            f.write(result)
-            f.close()
+            with open(options.outfile, 'w', encoding='utf-8') as outfile:
```

Comment:
please excuse me for replacing io.open from your latest commit, i can put it back in place, but i think since 2.7 its just an alias for open...